pydoctor/zopeinterface.py

```diff
@@ -221,13 +220,19 @@ def _handleAssignmentInClass(self,
                     section='zopeinterface')
         else:
             match = schema_prog.match(funcName)
+
             if match:
```

Comment:
```suggestion
            if schema_prog.match(funcName):
```
Since we no longer use the match object anywhere else, we don't need to save it to a local variable either.